pt/news/_posts/2019-04-01-ruby-2-4-6-released.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: news_post
+title: "Lnaçado Ruby 2.4.6"
+author: "usa"
+translator: "jcserracampos"
+date: 2019-04-01 06:00:00 +0000
+lang: pt
+---
+
+Foi lançado o Ruby 2.4.6.
+
+Esta versão inclui cerca de 20 correções de bugs depois das versões anteriores e também inclui várias correções de segurança.
+Por favor, verifique os tópicos abaixo para detalhes.
+
+* [Múltiplas vulnerabilidade em RubyGems](/en/news/2019/03/05/multiple-vulnerabilities-in-rubygems/)
+
+Veja o [commit log](https://github.com/ruby/ruby/compare/v2_4_5...v2_4_6) para detalhes.
+
+Depois desta versão, nós vamos terminar a fase de manutenção normal do Ruby 2.4
+e começará a fase de manutenção de segurança dele.
+Isto signifa que após o lançamento da 2.4.6 nós nunca traremos qualquer correção de bug
+para a 2.4, exceto correções de segurança.
+O período da fase de manutenção de segurança está programda para 1 ano.
+Ao fim deste período, o suporte oficial ao Ruby 2.4 terminará.
+Assim sendo, nós recomendamos que você comece a planejar para migrar para o Ruby 2.6 ou 2.5
+
+## Download
+
+* [https://cache.ruby-lang.org/pub/ruby/2.4/ruby-2.4.6.tar.bz2](https://cache.ruby-lang.org/pub/ruby/2.4/ruby-2.4.6.tar.bz2)
```

Comment:
Please follow https://github.com/ruby/www.ruby-lang.org/pull/2026 to fix the download section, thanks :-)

Reply:
I will amend asap.

Reply:
@JuanitoFatas, done